src/Pages/Uptime/Create/index.jsx

```
@@ -202,6 +202,10 @@ const CreateMonitor = () => {
 		}));
 	};
 
+	const handleAddNotification = () => {
+		console.log("Add notification clicked");
+	  };
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)